src/methods/integrity.js

```diff
@@ -98,6 +98,10 @@ export function TestIntegrityLevel(realm: Realm, O: ObjectValue, level: Integrit
   // 2. Assert: level is either "sealed" or "frozen".
   invariant(level === "sealed" || level === "frozen", "invalid level");
 
+  if (realm.react.immutableObjects.has(O)) {
+    return true;
```

Comment:
I may have been wrong about this. This function is used by `Object.isFrozen(obj)`. I'm not sure if we should return true for that since technically it is not runtime frozen. We should probably just move this check into Havoc at the same place as the TestIntegrityLevel call.

Reply:
Actually I wanted them to pass for this check as they are frozen if react options are enabled – just like they would be in the real world on DEV.